### PR TITLE
Prevent caching on the healthcheck (status) end-point

### PR DESCRIPTION
**Pull request by jon-betts**
For: https://github.com/hypothesis/product-backlog/issues/1198

This is just to add a single functest but none existed before, so some boilerplate had to be added. To get this going the unit tests were also nested in a directory to make room.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,6 +44,10 @@ checkformatting: python
 .PHONY: test
 test: backend-test frontend-test
 
+.PHONY: functests
+functests: python
+	@tox -qe functests
```
Comment (seanh) [inline Makefile:49]: We should add `functests` to `make sure` as well

File: requirements/functests.in [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-r tests.txt
```
Comment (seanh) [inline requirements/functests.in:1]:
```suggestion
-r requirements.txt
```

We don't generally want all unit test requirements to also be functest requirements (this is how the other projects do it)

**Review by seanh — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jon-betts) [reply Makefile:49]: Done

File: requirements/functests.in (hunk 2 above)
Comment (jon-betts) [reply requirements/functests.in]: Done

**Pull request merged**